.travis.yml

```diff
@@ -62,7 +62,8 @@ script:
   - if [ "$GOOS" = "linux" ]; then make coverage ; fi
   - if [ "$GOOS" = "linux" ]; then sudo PATH=$PATH GOPATH=$GOPATH make root-coverage ; fi
   - if [ "$GOOS" = "linux" ]; then sudo PATH=$PATH GOPATH=$GOPATH make integration ; fi
-  - if [ "$GOOS" = "linux" ]; then sudo PATH=$PATH GOPATH=$GOPATH make integration-parallel ; fi
+  # Run the integration suite a second time. See discussion in github.com/containerd/containerd/pull/1759
```

Comment:
This reverses the order of the parallel tests from before.

@mlaventure Does it matter in which order these get run to have the desired effect?

Reply:
It doesn't/shouldn't